source/index.ts

```diff
@@ -140,36 +59,38 @@ class Chromium {
     };
   }
 
-  /**
-   * Inflates the current version of Chromium and returns the path to the binary.
-   * If not running on AWS Lambda nor Google Cloud Functions, `null` is returned instead.
-   */
-  static get executablePath(): Promise<string> {
-    if (Chromium.headless !== true) {
-      return Promise.resolve(null);
-    }
-
-    if (existsSync('/tmp/chromium') === true) {
-      for (const file of readdirSync('/tmp')) {
+  static async prepare(folder: string) {
+    await fs.mkdir(folder, { recursive: true, mode: 0o777 })
+    const chromiumExpectedPath = join(folder, 'chromium')
+    if (await fileExists(chromiumExpectedPath)) {
```

Comment:
How will we do things in the agent? Will we always `prepare` to a fresh directory, or will we just do this once on startup?

I imagine that the public locations are normally using a fresh directory, and maybe this code path wasn't really being tried out before.

Reply:
Just asking because the unlinking `core.chromium` that they have seems interesting. I wonder what it does, and if there's anything else we need to do to keep chrome "fresh" between runs.